src/workflow/ArcanistDiffWorkflow.php

```diff
@@ -1861,6 +1861,29 @@ private function validateCommitMessage(
       }
     }
 
+    $config = $this->getConfigurationManager();
+    $mandatory_fields = $config->getConfigFromAnySource(
```

Comment:
Yes, it is an .arcconfig source. The check is off by default. Individual repositories can 
 - enable it by adding `"differential.mandatory_fields": ["testPlan", "revertPlan"]` to arcconfig.
 - disable it (if enabled gloably) by adding `"differential.mandatory_fields": []` to arcconfig.